appinventor/appengine/src/com/google/appinventor/server/BuildOutputServlet.java

```diff
@@ -118,4 +139,14 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOExc
     out.write(content);
     out.close();
   }
+
+  public RawFile getManifest(String host, Nonce nonce) throws IOException {
+    RawFile manifest = fileExporter.exportProjectOutputFile(nonce.getUserId(), nonce.getProjectId(), null, "manifest.plist");
+    String content = new String(manifest.getContent());
+    String base = "https://" + host + "/b/";
```

Comment:
Add a note here that iOS will only pull the file if it is over HTTPS, so we don't bother checking whether the original protocol was HTTP since that would already end in a failure state. Plus, we support ai2 over HTTPS now.